components/renderers/controls/autocomplete/autocomplete-renderer.ts

```diff
@@ -0,0 +1,23 @@
+///<reference path="../../../references.ts"/>
+
+class AutoCompleteControl implements JSONForms.IRenderer {
```

Comment:
Should be called AutoCompleteRenderer in order to conform with the new naming schema.